spec/stat_tracker_spec.rb

```diff
@@ -293,4 +284,42 @@
   it 'returns fewest_tackles' do
     expect(stat_tracker.fewest_tackles("20122013")).to eq("FC Dallas")
   end
+
+  it 'returns individual team info' do
+    expect(stat_tracker.team_info('29')).to eq({
+      'team_id' => '29',
+      'franchise_id' => '36',
+      'team_name' => "Orlando Pride",
+      'abbreviation' => 'FLP',
+      'link' => '/api/v1/teams/29'
+    })
+  end
+
+ it 'returns the best season' do
+   fake_data = [
+     {
+       :game_id => "2014030411",
+       :team_id => '16',
+       :result => "WIN"
+     },
+     {
+       :game_id => "2014030411",
+       :team_id => '14',
```

Comment:
we may not have needed an extra team in here since it is checking one team_id's data based on the season, but it also doesn't hurt to make sure our method doesn't get the wrong information.

Reply:
Negative! we must have my beloved team 14!